modules/service-accounts-default.adoc

```diff
@@ -60,6 +60,11 @@ Three service accounts are automatically created in each project:
 pushing images to any imagestream in the project using the internal Docker
 registry.
 
+[NOTE]
+====
+The `builder` service account is not created if the `Build` cluster capability is not enabled.
+====
+
```

Comment:
```suggestion
[NOTE]
====
The `builder` service account is created only if the `Build` cluster capability is enabled.
====
Maybe consider phrasing this sentence in positive rather than negative terms. In context, you're adding a note about service accounts that ARE created (rather than NOT created).
```

Reply:
Good point. As we chatted on slack about, I'll probably leave this as is for now because we have the same wording in there already for the DeploymentConfig capability, but might consider if it's worth updating them both in the future.